### ISSUE TITLE
limit(x/sin(1/x),x,0); wrong

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3619]
Imported from SourceForge on 2024-07-08 10:24:54
Created by **[macrakis](https://sourceforge.net/u/macrakis/)** on 2020-11-22 22:46:23
Original: https://sourceforge.net/p/maxima/bugs/3680

---

limit(x/sin(1/x),x,0) gives 0.
In fact, x/sin(1/x) can be made arbitrarily large arbitrarily close to x=0, so the correct limit is UND.
To see this, look at x= 1/(n*%pi+eps).

Maxima 5.43.0 SBCL 1.4.14 on Windows

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 10:24:55
Created by **[macrakis](https://sourceforge.net/u/macrakis/)** on 2020-11-22 22:47:25
Original: https://sourceforge.net/p/maxima/bugs/3680/#50e4

---

- Description has changed:

Diff:

~~~~

--- old
+++ new
@@ -1,3 +1,6 @@
 limit(x/sin(1/x),x,0) gives 0.
 In fact, x/sin(1/x) can be made arbitrarily large arbitrarily close to x=0, so the correct limit is UND.
 To see this, look at x= 1/(n*%pi+eps).
+
+Maxima 5.43.0 SBCL 1.4.14 on Windows
+

~~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 10:24:59
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2020-12-14 19:03:03
Original: https://sourceforge.net/p/maxima/bugs/3680/#738a

---

- **labels**:  --> limit

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 10:25:06
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2021-02-21 21:29:53
Original: https://sourceforge.net/p/maxima/bugs/3680/#7f1b

---

Fixed by [b173fdb7ae53fb2871ac7ff46942842dba4b3c2b]
